Messenger/Messenger.Core/Services/MessengerService.cs

```diff
@@ -444,13 +444,13 @@ public static async Task<bool> UpdateTeamName(string teamName, uint teamId)
         /// </summary>
         /// <param name="teamId">The id of the team to delete</param>
         /// <returns>True if the team was successfully deleted, false otherwise</returns>
-        public static async Task<bool> DeleteTeam(uint teamId)
+        public static async Task<Team> DeleteTeam(uint teamId)

```

Comment:
You should not return without considering the value of `result` since you do not know if everything went fine.

Reply:
Fixed in 09f420c.